src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/ItemsToSignComparer.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Maestro.Tasks
+{
+    /// <summary>
+    ///     Compares ItemsToSign objects based on file
+    /// </summary>
+    public class ItemsToSignComparer : IEqualityComparer<ItemsToSign>
+    {
+        public bool Equals(ItemsToSign x, ItemsToSign y)
+        {
+            return x.File.Equals(y.File, StringComparison.OrdinalIgnoreCase);
```

Comment:
This file is a subpath I assume? As in we wouldn't see any weirdness file layouts were different on two different machines?

Reply:
Yeah it is just the file name with no path at all.  `<ItemsToSign File="Microsoft.Cci.Extensions.5.0.0-ci.nupkg" />`